framework/analysis.py

```diff
@@ -222,8 +222,8 @@ def printprocess(self, i):
     nevents = self.nRunEvents
     i0 = self.firstEvent
     if int(i) % int(self.nEventsPrintOut) == 0:
-      if   self.index == -1: print 'Processing... %i / %i (%1.2f'%(i,nevents, (float(i)-i0)/nevents*100) + '%)'
-      elif self.index == 0:  print 'Processing... %1.2f'%((float(i)-i0)/nevents*100) + ' %'
+      print 'Processing... %i / %i (%1.2f'%(i,nevents, (float(i)-i0)/nevents*100) + '%)'
```

Comment:
Please

Reply:
Restored to original less verbose syntax